docs/Env_Setup.rst

```diff
@@ -108,6 +108,188 @@ with a minimal fedora 23 installation. It will leave any logs under the `logs`
 directory and any generated artifacts under `exported-artifacts`.
 
 
-
    .. _`official yapf repo`: https://github.com/google/yapf
    .. _`copr`: https://copr.Fedoraproject.org/coprs/fulltext/?fulltext=yapf
+
+
+Running Lago in virtualenv
+---------------------------
+This was tested on a fresh Fedora-24 Cloud image, but should work on Fedora 23
+and most likely with little package-names resolving on CentOS >7.
+
+Install dependencies::
+
+  > sudo dnf install -y redhat-rpm-config libvirt libvirt-devel \
+     libguestfs-devel libguestfs-tools qemu-img qemu-kvm \
+     libvirt-daemon git gcc libffi-devel python-devel \
+     openssl-devel bats python-magic python-libguestfs
+
+Note that ``python-magic`` and ``python-libguestfs`` are currently not
+available on PyPI, so we have to install them from RPMs.
+
+Install virtualenv::
+
+  > sudo pip install virtualenv
+
+Add your user to ``qemu`` group and vice versa::
+
+  > sudo usermod -a -G qemu "$USER"
+  > sudo usermod -a -G "$USER" qemu
```

Comment:
An alternative to adding `qemu` to `$USER` group can be to use file access control lists (I am using `/var/lago` instead of `$HOME/lago/data`).